### ISSUE TITLE
Light weight prediction server (other than tf serving) for 1.25.3

### ISSUE THREAD

**Issue by Morizeyao** [OpenNMT/OpenNMT-tf#588]
Hi there, thank you for helping me dealing with [problems in distributed training](https://github.com/OpenNMT/OpenNMT-tf/issues/582). With your help I have trained a model that works quite well on validation set.

Now I need to make a demo of my translation model. But I don't have the environment to run a [tf serving docker service](https://github.com/OpenNMT/OpenNMT-tf/blob/r1/docs/serving.md).

Is there a way or some sample code showing how I can make a simple http demo server (flask style should be good enough) of a opennmt-tf model?

I am familiar with Pytorch but not very good at TensorFlow 1.0. Since TensorFlow has better Hadoop support I chose opennmt-tf 1.25.3 for this project.

Thank you very much!

**Comment by guillaumekln** [comment]
If you trained a standard Transformer model, can you check if [CTranslate2](https://github.com/OpenNMT/CTranslate2) can work for you? It does not provide a ready-to-use server but it should not be too complex to wrap the translation API.

Otherwise, consider upgrading to TensorFlow 2. It is now [easier](https://github.com/OpenNMT/OpenNMT-tf/tree/master/examples/serving/python) to run a TensorFlow model in Python.

**Comment by Morizeyao** [comment]
Thank you for your quick response! Yes, I am training a Transformer model so I will check CTranslate2. I will also try to upgrade to TF2. :)

Will reopen this if I have made any progress making a lightweight server.
